eZ/Bundle/EzPublishCoreBundle/Imagine/PlaceholderProvider.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Imagine;
+
+use eZ\Publish\Core\FieldType\Image\Value as ImageValue;
+
+interface PlaceholderProvider
+{
+    /**
+     * Provides replace for specified image.
```

Comment:
What about "Provides a placeholder image path for a given Image FieldType value".

Reply:
Sound better, thx